src/devtools/client/debugger/src/components/Editor/EditorPane.tsx

```diff
@@ -27,9 +27,9 @@ export const EditorPane = () => {
   useLayoutEffect(() => {
     const root = document.querySelector<HTMLElement>(":root")!;
     if (enableLargeText) {
-      root.style.setProperty("--theme-code-font-size", "14px");
+      root.style.setProperty("--font-size-regular-monospace", "14px");
     } else {
-      root.style.setProperty("--theme-code-font-size", "11px");
+      root.style.setProperty("--font-size-regular-monospace", "11px");
```

Comment:
I imagine we can delete this entire effect, since these values are already defined at the root.

Reply:
Adjusted